README.md

```diff
@@ -7,11 +7,15 @@ https://assetstore.unity.com/packages/essentials/tutorial-projects/endless-runne
 The tests are meant to be run on an Windows or MacOS device. The app is provided in this repository, under the mac/  and windows/ folder.
```

Comment:
Delete `The app is provided in this repository, under the mac/  and windows/ folder. ` after the links for builds are provided